CONTRIBUTING.md

```diff
@@ -112,13 +112,15 @@ Testing Lints
 **Creating Unit Tests.** Every lint should also have corresponding unit
 tests (generally at least one for a success and one for a failure condition). We
 have typically generated test certificates using Go (see
-[documentation][CreateCertificates] for details), but OpenSSL
-could also be used. Test certificates should be placed in `testdata/` and called
-from the test file created by `newLint.sh`. You may want to prepend the PEM with
-the output of `openssl x509 -text`. You can run your lint against a test
-certificate from a unit test using the `test.TestLint` helper function.
+[documentation][CreateCertificates] for details and [example][certGenerator] as
```

Comment:
couple nit:

* it's `CreateCertificate` singular,
* also I think sensible to reference the namespace, `x509.CreateCertificate`
* lastly, a grammatical tweak (and example as reference -> and this example code as reference).


```suggestion
[documentation][x509.CreateCertificate] for details and [this example code][certGenerator] as
```